internal/rest/avi_obj_vsvip.go

```diff
@@ -128,6 +129,26 @@ func (rest *RestOperations) AviVsVipBuild(vsvip_meta *nodes.AviVSVIPNode, vsCach
 					if vsvip_meta.VipNetworks[0].V6Cidr != "" {
 						lib.UpdateV6(vip, &vsvip_meta.VipNetworks[0])
 					}
+					if lib.GetCloudType() == lib.CLOUD_NSXT &&
```

Comment:
Seems like same code duplicated at 2 places, can you make a function or something out of it

Reply:
removed duplication